api/api/controllers/agents_controller.py

```diff
@@ -59,8 +58,8 @@ def delete_agents(pretty=False, wait_for_complete=False, list_agents='all', purg
 
 @exception_handler
 def get_all_agents(pretty=False, wait_for_complete=False, offset=0, limit=None, select=None, sort=None, search=None,
-                   status=None, q='', older_than=None, os_platform=None, os_version=None, os_name=None, manager=None,
-                   version=None, group=None, node_name=None, name=None, ip=None, registerIP=None):
+                   status=None, q='', older_than=None, manager=None, version=None, group=None, node_name=None,
+                   name=None, ip=None, registerip=None):
```

Comment:
`registerip` should be `register_ip`. I would do that change and capturing the param from the `connexion.request.args` instead of declaring the param in the `get_all_agents` function.

Reply:
Done